kubernetes/thelounge.yml.erb

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: service
+spec:
+  selector:
+    app: thelounge
+  ports:
+    - port: 80
+      targetPort: 9000
+--
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: deployment
+  labels:
+    app: thelounge
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: thelounge
+  template:
+    metadata:
+      labels:
+        app: thelounge
+    spec:
+      containers:
+        - name: thelounge
+          image: "tonylianlong/thelounge:1.0.0"
+          resources:
+            limits:
+              memory: 2048Mi
```

Comment:
this seems like... a lot. Currently thelounge is runing on 512Mi (although with a whole CPU instead of 750m). See https://github.com/ocf/services/blob/master/apps/thelounge/app.yaml#L5.

Reply:
Then I could change it to something like 750Mi?